src/components/editor/component.tsx

```diff
@@ -232,6 +232,20 @@ export class EditorComponentBase<TOutput, TDoc extends EditorDocument> {
 
         return (
             <>
+                {/* content warnings */}
```

Comment:
I would prefer to keep this positioned towards the bottom (looks like cohost had it underneath tags). Changing it to be always visible is fine.

If you feel strongly about moving it above the post box, we can discuss further.

Reply:
Sounds good, lmk if you want any pointers about how settings work in pillbug.